book/09-git-and-other-scms/sections/client-bzr.asc

```diff
@@ -128,9 +128,9 @@ $ git push origin master
 Git's remote-helpers framework has some limitations that apply.
 In particular, these commands don't work:
 
-* git push origin :branch-to-delete (Bazaar can't accept ref deletions in this way.)
-* git push origin old:new (it will push 'old')
-* git push --dry-run origin branch (it will push)
+* git push origin :branch-to-delete (Bazaar can't accept ref deletions in this way).
+* git push origin old:new (it will push 'old').
+* git push --dry-run origin branch (it will push).
```

Comment:
Bullet lists don't need punctuation.

Reply:
How about this?

```
* `git push origin :branch-to-delete`: Bazaar can't accept ref deletions in this way,
* `git push origin old:new`: it will push 'old',
* `git push --dry-run origin branch`: it will push.
```

* `git push origin :branch-to-delete`: Bazaar can't accept ref deletions in this way,
* `git push origin old:new`: it will push 'old',
* `git push --dry-run origin branch`: it will push.

see APA style section 6.52 (Items That Contain Both Phrases and Sentences).

Reply:
I don't think that it's the best way to use the in-line comment for the command example.
The colon symbol is used in Git as a special symbol and the additional one might confuse people who read the book.

Possibly, the better way is to use `//` to separate command and a comment.

Reply:
@Morganov,

> the better way is to use // to separate command and a comment.

How about using `#` to separate command and a comment instead of `//`? Even if user copy whole command, they won't get any error because in most shell `#` sign is used for comment.